libtextsecure/helpers.js

```diff
@@ -90,6 +90,11 @@ window.textsecure.utils = function() {
         return number.split(".");
     };
 
+    self.isNumberSane = function(number) {
```

Comment:
I don't think libtextsecure should be handling number validation because a) it's extra work, b) none of the functionality of the library really depends on having valid numbers afaik, c) eventually we want to have non-number identifiers, and so might consumers of the lib.

In general, I would advocate the use of `members` and/or `identifiers` rather than number wherever applicable.